tests/api/circulation/test_loan_bulk_extend.py

```diff
@@ -22,32 +22,45 @@
 from tests.api.conftest import _create_records
 from tests.helpers import user_login
 
-pid_start_value = 99
-amount_of_ongoing_loans = 5
-pid_end_value = pid_start_value + amount_of_ongoing_loans
+pid_start_value = 200
+amount_of_ongoing_loans = 3
+amount_of_overdue_loans = 3
+pid_end_value = \
+    pid_start_value + amount_of_ongoing_loans + amount_of_overdue_loans
 
 
 def create_loans(start=pid_start_value, end=pid_end_value):
     ongoing_loans = []
+    overdue_loans = []
 
     day_range_loan_extendable = \
         current_app.config["ILS_CIRCULATION_LOAN_WILL_EXPIRE_DAYS"]
 
     now = arrow.utcnow()
-    end_date_not_expired = now + timedelta(days=day_range_loan_extendable - 1)
 
-    def loan_generator(pid_value):
+    start_date_expiring = now.isoformat()
+    start_date_overdue = (now - timedelta(days=5)).isoformat()
+
+    end_date_expiring = \
+        (now + timedelta(days=day_range_loan_extendable - 1)).isoformat()
+    end_date_overdue = (now - timedelta(days=2)).isoformat()
```

Comment:
```suggestion
    end_date_overdue = (now - timedelta(days=<CONFIG VAR>)).isoformat()
```
I would use config variable for this, because if it changes, the test will be incorrect

Reply:
An overdue loans is always going to be simply before the current date i would assume :thinking:

Reply:
ah I see, I misunderstood the intention here